### PR TITLE
Upgrade Grafana

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Upgrade Grafana

**Related issue(s)**

See #8750

### CODE REVIEW

File: resources/serverless/templates/function-dashboard.yaml [hunk 1]
```diff
@@ -1084,7 +1084,9 @@ data:
       "refresh": "10s",
       "schemaVersion": 25,
       "style": "dark",
-      "tags": [],
```
Comment (rakesh-garimella) [inline resources/serverless/templates/function-dashboard.yaml:1087]: do we need tags for other dashboards too ?

Comment (suleymanakbas91) [reply resources/serverless/templates/function-dashboard.yaml:1087]: Yes, we already have tags for all the dashboards. This dashboard had a syntax problem. That's why we were not able to see it on grafana and not realized that it had a missing tag.

**Pull request merged**